src/app/games/lasca/LascaTutorial.ts

```diff
@@ -0,0 +1,78 @@
+            LascaState.getInitialState(),
+        ),
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`The goal of Lasca is, like for draughts, to render the opponent out of possible move, either by capturing all his pieces, either by blocking them.`,
+            LascaState.getInitialState(),
+        ),
+        // 3. Simple step
+        TutorialStep.anyMove(
+            $localize`Steps`,
+            $localize`A simple step is made by one diagonal move forward, left or right. Click on the chosen piece, then on it's landing square.<br/><br/>You are playing Dark, do the first move.`,
+            LascaState.getInitialState(),
+            LascaMove.fromStep(new Coord(4, 4), new Coord(3, 3)).get(),
+            $localize`Congratulations!`,
+        ),
+        // 4. Simple capture
+        TutorialStep.anyMove(
+            $localize`Capture`,
+            $localize`You have to capture when you can. It is the case here, so click on the piece that must capture, and then on its landing square.<br/><br/>You're playing Dark, go ahead.`,
+            LascaState.from([
+                [_v, __, __, __, _v, __, _v],
+                [__, __, __, _v, __, _v, __],
+                [__, __, _v, __, _v, __, _v],
+                [__, _v, __, __, __, __, __],
+                [_u, __, _u, __, _u, __, _u],
+                [__, _u, __, _u, __, _u, __],
+                [_u, __, _u, __, _u, __, _u],
+            ], 2).get(),
+            LascaMove.fromCapture([new Coord(2, 4), new Coord(0, 2)]).get(),
+            $localize`Congratulations, notice that the captured piece was not removed from the board, but put below the capturing pieces.`,
+        ),
+        // 5. Longer capture
+        TutorialStep.anyMove(
+            $localize`Multiple captures (1/2)`,
+            $localize`If, after the beginning of your capture, the piece that just moved can capture another piece, it has to capture until she can no longer. To do so, you must then click again on the next landing space. Note that, when capturing with an officer, you cannot jump twice over the same stack.<br/><br/>You are playing Dark, a double capture is possible, go ahead.`,
+            LascaState.from([
+                [_v, __, __, __, _v, __, _v],
+                [__, _v, __, _v, __, _v, __],
+                [__, __, _v, __, _v, __, _v],
+                [__, _v, __, __, __, __, __],
+                [__, __, _u, __, _u, __, _u],
+                [__, __, __, _u, __, _u, __],
+                [_u, __, __, __, _u, __, _u],
+            ], 2).get(),
+            LascaMove.fromCapture([new Coord(2, 4), new Coord(0, 2), new Coord(2, 0)]).get(),
+            $localize`Congratulations, you captured two pieces, and, by reaching the last line, your piece has now been promoted to the rank of officer ! Now, it can move and capture backward, which is illegal for normal piece.`,
```

Comment:
too, much, commas (remplace le , and, par un .)

Reply:
Tu veux peut être dire que tout les whitch doivent être précédés d'une virgule ?
C'toi qu'est dans l'coma !
J'ai 66 which précédés d'une virgule dans meugueuxpeu

Reply:
nvm y'a bien un comma avant une sorcière !
Peut-être changer le début de la deuxième phrase en : "It can now move ..."

Reply:
hors de date, on a débattu plus haut que ça devait devenir deux steps, celui-ci n'existe plus !
Tu PéRou !